keras_retinanet/bin/debug.py

```diff
@@ -118,7 +118,7 @@ def csv_list(string):
     oid_parser = subparsers.add_parser('oid')
     oid_parser.add_argument('main_dir', help='Path to dataset directory.')
     oid_parser.add_argument('subset', help='Argument for loading a subset from train/validation/test.')
-    oid_parser.add_argument('--version',  help='The current dataset version is V3.', default='2017_11')
+    oid_parser.add_argument('--version',  help='The current dataset version is V4.', default='2018_04')
```

Comment:
Isn't it a bit misleading that the help says `'V4'`, but the default is `'2018_04'`?

Reply:
Yes. You are right. I'll use only the short label and behind the scene build the right path.